src/main/java/us/ihmc/euclid/tools/EuclidCoreRandomTools.java

```diff
@@ -1556,11 +1555,9 @@ public static void randomizeAxisAngle(Random random, AxisAngleBasics axisAngleTo
     */
    public static void randomizeAxisAngle(Random random, double minMaxAngle, AxisAngleBasics axisAngleToRandomize)
    {
-      // Generate uniformly random point on unit sphere (based on http://mathworld.wolfram.com/SpherePointPicking.html )
-      double height = 2.0 * random.nextDouble() - 1.0;
+      // Generate uniformly random point on the unit-sphere (based on http://mathworld.wolfram.com/SpherePointPicking.html )
       double angle = nextDouble(random, minMaxAngle);
-      double radius = EuclidCoreTools.squareRoot(1.0 - height * height);
-      axisAngleToRandomize.set(radius * EuclidCoreTools.cos(angle), radius * EuclidCoreTools.sin(angle), height, angle);
+      axisAngleToRandomize.set(EuclidCoreRandomTools.nextVector3D(random), angle);
```

Comment:
The old implementation was incorrect in the sense that it wouldn't generate random axes that would be uniformly distributed.
Fixing this seems to break other tests. I'll work on fixing them